man/systemd-system.conf.xml

```diff
@@ -349,25 +349,20 @@
         <term><varname>DefaultTimeoutAbortSec=</varname></term>
         <term><varname>DefaultRestartSec=</varname></term>
 
-        <listitem><para>Configures the default timeouts for starting,
-        stopping and aborting of units, as well as the default time to sleep
-        between automatic restarts of units, as configured per-unit in
-        <varname>TimeoutStartSec=</varname>,
-        <varname>TimeoutStopSec=</varname>,
-        <varname>TimeoutAbortSec=</varname> and
-        <varname>RestartSec=</varname> (for services, see
+        <listitem><para>Configures the default timeouts for starting, stopping and aborting of units, as well
+        as the default time to sleep between automatic restarts of units, as configured per-unit in
+        <varname>TimeoutStartSec=</varname>, <varname>TimeoutStopSec=</varname>,
+        <varname>TimeoutAbortSec=</varname> and <varname>RestartSec=</varname> (for services, see
```

Comment:
It's not just that there are no content changes at all in this particular section of the diff (tho there aren't), the bigger concern is that the _old_ formatting was actually **better** and more diff-friendly.